docs/scos/dev/migration-concepts/migrate-to-decoupled-glue-infrastructure/decoupled-glue-infrastructure-integrate-api-key-authorization.md

```diff
@@ -0,0 +1,170 @@
+| AuthorizationResponse  | class | created | src/Generated/Shared/Transfer/AuthorizationResponseTransfer |
+
+{% endinfo_block %}
+
+### 3) Set up behavior
+
+1. Activate the following plugins:
+
+| PLUGIN | SPECIFICATION                                | NAMESPACE |
+| --- |----------------------------------------------| --- |
+| ApiKeyAuthorizationRequestExpanderPlugin | Expands the request by the API Key provided. | Spryker\Glue\ApiKeyAuthorizationConnector\Plugin\GlueBackendApiApplicationAuthorizationConnector\ApiKeyAuthorizationRequestExpanderPlugin |
+| ApiKeyAuthorizationStrategyPlugin | Executes the API Key verification process.         | Spryker\Zed\ApiKeyAuthorizationConnector\Communication\Plugin\Authorization\ApiKeyAuthorizationStrategyPlugin |
+
+<details open>
+<summary markdown='span'>src/Pyz/Glue/GlueBackendApiApplicationAuthorizationConnector/GlueBackendApiApplicationAuthorizationConnectorDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Glue\GlueBackendApiApplicationAuthorizationConnector;
+
+use Spryker\Glue\ApiKeyAuthorizationConnector\Plugin\GlueBackendApiApplicationAuthorizationConnector\ApiKeyAuthorizationRequestExpanderPlugin;
+use Spryker\Glue\GlueBackendApiApplicationAuthorizationConnector\GlueBackendApiApplicationAuthorizationConnectorDependencyProvider as SprykerGlueBackendApiApplicationAuthorizationConnectorDependencyProvider;
+
+class GlueBackendApiApplicationAuthorizationConnectorDependencyProvider extends SprykerGlueBackendApiApplicationAuthorizationConnectorDependencyProvider
+{
+    protected function getAuthorizationRequestExpanderPlugins(): array
+    {
+        return [
+            new ApiKeyAuthorizationRequestExpanderPlugin(),
+        ];
+    }
+}
+
+```
+</details>
+
+<details open>
+<summary markdown='span'>src/Pyz/Zed/Authorization/AuthorizationDependencyProvider.php</summary>
+
+```php  
+<?php
+
+namespace Pyz\Zed\Authorization;
+
+use Spryker\Zed\ApiKeyAuthorizationConnector\Communication\Plugin\Authorization\ApiKeyAuthorizationStrategyPlugin;
+use Spryker\Zed\Authorization\AuthorizationDependencyProvider as SprykerAuthorizationDependencyProvider;
+
+class AuthorizationDependencyProvider extends SprykerAuthorizationDependencyProvider
+{
+    protected function getAuthorizationStrategyPlugins(): array
+    {
+        return [
+            new ApiKeyAuthorizationStrategyPlugin(),
+        ];
+    }
+}
+
+```
+</details>
+
+2. Add the following configuration:
+
+<details open>
+<summary markdown='span'>src/Pyz/Zed/Authorization/AuthorizationConfig.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\Authorization;
+
+ use Spryker\Zed\Authorization\AuthorizationConfig as SprykerAuthorizationConfig;
+
+ class AuthorizationConfig extends SprykerAuthorizationConfig
+ {
+     /**
+      * {@inheritDoc}
+      *
+      * @return bool
+      */
+     public function isMultistrategyAuthorizationAllowed(): bool
```

Comment:
When the configuration section is added (requested in another comment), we'll have a table of configurations that we add + code samples, so, in the table, we should have a description of why we need this code.